ci-operator/config/openshift-eng/ocp-qe-perfscale-ci/openshift-eng-ocp-qe-perfscale-ci-main__metal-4.20-nightly-x86.yaml

```diff
@@ -0,0 +1,163 @@
+base_images:
+  cerberus:
+    name: cerberus
+    namespace: chaos
+    tag: cerberus-prow
+  ocp-qe-perfscale-ci:
+    name: ocp-qe-perfscale-ci
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.20"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: daily-virt-6nodes
+  capabilities:
+  - intranet
+  cron: 30 0 * * *
```

Comment:
I think you need to change the cron schedule. It's the same as in 4.19. Also, we don't support cron apart from the y-stream release, 4.20, right now. I suggest removing cron from 4.19 and previous versions because baremetal profile falls under the priority 2 - manual-triggered category as per the PerfScale Prow job Rules.